packages/react-components/react-storybook-addon/src/theme.ts

```diff
@@ -8,12 +8,18 @@ import {
 
 import type { Theme } from '@fluentui/react-theme';
 
+export const WEB_LIGHT = 'web-light';
```

Comment:
ideally these constants shouldn't be public API , see https://github.com/microsoft/fluentui/pull/25162/files#r998423431

Reply:
These have been removed